cookiecutter/generate.py

```diff
@@ -10,6 +10,7 @@
 from __future__ import unicode_literals
 import logging
 import os
+import os.path as osp
```

Comment:
Sorry, just an idiom I use frequently.  I'm lazy and os.path.... feels to long.  Just using "path" can be probelematic because I've found people like to use path as a variable name and I've encountered a few weird bugs in the past due to that naming collision.